v2.0/delete.md

```diff
@@ -6,6 +6,8 @@ toc: false
 
 The `DELETE` [statement](sql-statements.html) deletes rows from a table.
 
+{{site.data.alerts.callout_danger}}<code>CASCADE</code> does not list objects it drops or updates, so it should be used cautiously. If you delete a row that is referenced by a <a href="foreign-key.html">foreign key</a> and <code>ON DELETE CASCADE</code> is set, all of the dependent rows will also be deleted.{{site.data.alerts.end}}
```

Comment:
This seems abrupt to me - as a user, I would expect to first find information about how to use `DELETE` in normal scenarios, and then be warned about potential problems. Can this warning be moved to another part of the document, may be with an example of using `CASCADE`?